### ISSUE TITLE
The Windows CI fails due to timeout.

### ISSUE THREAD

**Issue by HideakiImamura** [optuna/optuna#5099]
### Expected behavior

I have been observing an issue for the past few days where the Windows CI randomly fails due to timeout, regardless of whether it's a PR or on schedule. It looks something like the following CI:　https://github.com/optuna/optuna/actions/runs/6777581250/job/18421473256#step:8:15

All of the CIs appear to fail on the first test case of the visualization features using matplotlib. I have no windows machine to reproduce the error on my local environment.

### Environment

- Optuna version: the current master https://github.com/optuna/optuna/commit/523dcc276a64e0d22ff8c92c17b88509b94bea88
- Python version: 3.9
- OS: Windows Server 2022
- (Optional) Other libraries and their versions:

### Error messages, stack traces, or logs
Here is the CI's output.

```shell

Run pytest -m "not integration"
============================= test session starts =============================
platform win32 -- Python 3.9.13, pytest-7.4.3, pluggy-1.3.0
rootdir: D:\a\optuna\optuna
configfile: pyproject.toml
plugins: jaxtyping-0.2.23, typeguard-2.13.3
collected 4850 items / 839 deselected / 4011 selected

tests\test_callbacks.py .                                                [  0%]
tests\test_cli.py ...................................................... [  1%]
...
tests\trial_tests\test_trial.py ........................................ [ 80%]
........................................................................ [ 81%]
..............................                                           [ 82%]
tests\trial_tests\test_trials.py ....................................... [ 83%]
...                                                                      [ 83%]
tests\visualization_tests\test_contour.py ..
Error: The operation was canceled.
```

### Steps to reproduce

Trigger the windows CI by workflow dispatcher on your fork.

**Comment by not522** [comment]
CI seems to be stuck on the third test in `test_contour.py`. This is the first test in `test_plot_contour`, which outputs an image using plotly. I am not sure, but perhaps this issue https://github.com/plotly/Kaleido/issues/110 is related.

**Comment by HideakiImamura** [comment]
Downgrading kaleido to 0.1.0 will resolve issue according to the comments like https://github.com/plotly/Kaleido/issues/110#issuecomment-1520317283. I will hotfix it.

**Comment by not522** [comment]
According to the issue, 0.1.0post1 may be better.

**Comment by not522** [comment]
Unfortunately, kaleido 0.1.0post1 did not resolve this issue. https://github.com/optuna/optuna/actions/runs/6818449794

**Comment by not522** [comment]
In my understanding, there were two problems related to this issue, `test_contour.py` and `test_journal.py`. The first one was resolved by Kaleido 0.1.0post1. The second one has not been reproduced in these two months. I think that it is OK to close this issue. If this issue will be reproduced or you have some concerns, please feel free to reopen it.

**Comment by not522** [comment]
I reopen this issue since the problem with `test_journal.py` recurred. https://github.com/optuna/optuna/actions/runs/8530596405
